setup.py

```diff
@@ -45,11 +45,10 @@
       tests_require=['pytest', 'mock'],
       python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
I think you'd want to update `python_requires` as well, otherwise as soon as you add Python code that is not Py2-compatible, you will break Python 2 users who don't set an upper bound on your dependency (`gym>=0.10`), or expect compatibility w.r.t. semantic versioning (`gym>=0.15, <1.0` for example)  - note that this is a minor version change.

Reply:
good point, thanks!

Reply:
https://github.com/openai/gym/pull/1827

Reply:
Hey @cclauss , I've seen your drive-by comments on quite a few projects re: Python 3 migration. FYI - this is a common error when dropping Py2. Another project I saw this: https://github.com/hamcrest/PyHamcrest/issues/131. Perhaps this is something to keep an eye on in the projects that you come across.

Reply:
I would use the [`python_requires=">=3.5"`](https://github.com/psf/black/blob/master/setup.py#L64) approach when dropping legacy Python.

Reply:
Yeah, fully agree; I meant that more projects should be following this pattern, and I thought perhaps you are in the position to raise awareness of the proper way to drop Py2 support in other projects.